<launch>
  <arg name="points_topic" default="points_raw" />
  <arg name="output_topic" default="filtered_points" />
  <arg name="x_min" default="-50.0" />
  <arg name="x_max" default="50.0" />
  <arg name="y_min" default="-4.5" />
  <arg name="y_max" default="4.5" />

  <node pkg="lc_processor" name="lc_processor" type="lc_processor">
    <param name="points_topic" value="$(arg points_topic)" />
    <param name="output_topic" value="$(arg output_topic)" />
    <param name="x_min" value="$(arg x_min)" />
    <param name="x_max" value="$(arg x_max)" />
    <param name="y_min" value="$(arg y_min)" />
    <param name="y_max" value="$(arg y_max)" />
  </node>
</launch>
